mw-webapp/src/component/button/Button.module.scss

```diff
@@ -1,14 +1,21 @@
 @import "src/styles/variables";
 
 .button {
-  width: 100px;
-  height: 40px;
+  padding: 13px 17px;
+  border: none;
+  border-radius: $primaryBorderRadius;
+  background-color: $inputInitial;
```

Comment:
background for button can't has "input" color

Reply:
Checkbox has the same color. I have trouble with naming the bg-color for button and checkbox. So I think that button and checkbox are input types. I can't decide on an appropriate name.

Reply:
let's call it primatyColor for now

Reply:
Done